### PR TITLE
Generating Power Set Problem in Python

### PR DESCRIPTION
**Pull request by harsha-sam**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [☑️] I've read the contribution guidelines.
- [☑️] I've checked the issue list before deciding what to submit.
- [☑️] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Related Issue: #5899 
Related Pull Request: #5901

### CODE REVIEW

**Comment by harsha-sam** [comment]
https://github.com/TesseractCoding/NeoAlgo/pull/5901#pullrequestreview-638636556

### Regarding the multi-line comment requested change:

I would like to mention about how Multi-Line Comments in Python are not same as compared to  any other programming language. Python doesn't support multi-line comment blocks out of the box. The recommended way to comment out multiple lines of code in python is to use consecutive `# single-line comments`. [Python's PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/#block-comments) also favours repeated single-line comments. 

Checkout this program:
https://replit.com/join/fbvtsqxl-harshasam

In the above program, I used  """triple-quote strings""" to comment between an `if-else` block. This program raises a `SyntaxError` because triple quote strings are treated as regular strings with the exception that they can span multiple lines. These regular strings are not assigned to a variable and they will be garbage collected immediately as soon as the code executes. So, these are not ignored by the interpreter in the same way that  `# a single line comment` is.

There's a case that whenever these  """triple-quote strings""" are placed right after a function or class or top of a module, they are treated as `doc strings` and available at `obj.__doc__`

To fix the syntax error in the above program. Replace """the triple quote strings""" between the if-else block with `# single-line comments`. 

Hence, the better way to comment out multiple lines is to use consecutive `# single-line comments`.

File: Python/cp/Generating_Power_Set.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+# Sample Input 3:(Empty)
+# Enter values seperated by space:
+# Sample Output 3:
+# Power Set: [[]]
+
+# Sample Input 4:
+# Enter values seperated by space: a b
+# Sample Output 4
+# Power Set: [[], ['b'], ['a'], ['a', 'b']]
```
Comment (ankitaggarwal23) [inline Python/cp/Generating_Power_Set.py]: remove these extra test case

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/cp/Generating_Power_Set.py (hunk 1 above)
Comment (harsha-sam) [reply Python/cp/Generating_Power_Set.py]: Done

File: Python/cp/Generating_Power_Set.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+# Sample Output 1:
+# Power Set: [[], [3], [2], [2, 3], [1], [1, 3], [1, 2], [1, 2, 3]]
+# Sample Input 2:
+# Enter values seperated by space: 1 2 -1 0
+# Sample Output 2:
+# Power Set: [[], [0], [-1], [-1, 0], [2], [2, 0], [2, -1], [2, -1, 0],
+# [1], [1, 0], [1, -1], [1, -1, 0], [1, 2], [1, 2, 0],
+# [1, 2, -1], [1, 2, -1, 0]]
+# Time Complexity: O(N * 2*N)
+# Space Complexity: O(2*N)
```
Comment (ankitaggarwal23) [inline Python/cp/Generating_Power_Set.py]: maintain proper newline spacing wherever needed

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: Python/cp/Generating_Power_Set.py (hunk 2 above)
Comment (harsha-sam) [reply Python/cp/Generating_Power_Set.py]: Sure

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**